src/test/java/com/github/iveina2021/api_test/CreateAccountTest.java

```diff
@@ -0,0 +1,52 @@
+package com.github.iveina2021.api_test;
```

Comment:
It's really good that you want to separate set of the tests using different packages. But let's keep code conventions.

https://comaqa.gitbook.io/java-automation/stil-napisaniya-koda/konvenciya-imenovaniya

Reply:
Edited it